packages/swingset/src/cli/commands/bootstrap.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs'
+import { error, LOGS } from '../logs'
+import { Bootstrap } from '../types'
+import { detect, PM } from 'detect-package-manager'
+import childProcess from 'child_process'
```

Comment:
Convention is usually `child_process` to match package name but I used camelCasing for consistency